WebRoot/js/zimbraMail/core/ZmZimbraMail.js

```diff
@@ -2151,7 +2151,7 @@ function(view) {
 				var stb = appCtxt.getSearchController().getSearchToolbar();
 				if (appCtxt.get(ZmSetting.SHOW_SEARCH_STRING) && stb) {
 					var value = currentSearch ? currentSearch.query : app.currentQuery;
-					value = appName === ZmApp.SEARCH ? "" : value;
+					value = appName === ZmApp.SEARCH ? "" : (value || "");
```

Comment:
@k-kato : This change might lead to a side-effect of showing a blank space instead of the placeholder text while coming to 'Calendar' or 'Preferences' tab. This can be done safely by replacing next line of code with `stb.setSearchFieldValue(value ? value + " " : "");`

Reply:
@tarusharora thank you for your comment. The fix on https://github.com/Zimbra/zm-web-client/pull/392 is better.